cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/common/ETLUtils.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.templates.etl.common;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import com.google.common.base.Throwables;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * An Utility class for ETL
+ */
+public class ETLUtils {
+
+  /**
+   * Parses a frequency String to its long value
+   *
+   * @param frequencyStr the frequency string (ex: 5m, 5h etc).
+   * @return long equaivalent of frequency string
```

Comment:
nit: typo: equivalent.  Also, should be more descriptive, something like "milliseconds equivalent of the frequency string"